docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM 		ubuntu:16.04
```

Comment:
These large tabs should be reduced to a single space, I have never seen a Dockerfile formatted this way.